packages/ui/src/InteractiveList.tsx

```diff
@@ -112,8 +120,10 @@ export const InteractiveList = <V,>({
           kind="transparent"
           ariaLabel="Add Icon"
           icon={Plus}
-          className={styles.addIcon}
+          className={cn(styles.addIcon, { [styles.disabledIcon]: disabled })}
           size="medium"
+          disabled={!!disabled}
```

Comment:
`disabled={disabled}`

Reply:
I know, but it is for a type check. `disabled: boolean | undefined`